stable/redis/templates/redis-statefulset.yaml

```diff
@@ -0,0 +1,325 @@
+{{- if (include "redis.createStatefulSet" .) }}
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: {{ template "redis.fullname" . }}
+  labels: {{- include "redis.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels: {{- include "redis.matchLabels" . | nindent 6 }}
+  {{- if .Values.updateStrategy }}
+  strategy: {{ toYaml .Values.updateStrategy | nindent 4 }}
+  {{- end }}
+{{ include "redis.replicaCount" . | nindent 2}}
+  {{- if and .Values.cluster.enabled (not .Values.cluster.externalAccess.enabled) }}
+  serviceName: {{ template "redis.fullname" . }}-headless
+  {{- end }}
+  template:
+    metadata:
+      labels: {{- include "redis.labels" . | nindent 8 }}
+{{- if and .Values.metrics.enabled .Values.metrics.podLabels }}
```

Comment:
As much as we can, we try indenting the templates so it is more readable. For example, this could be

```
  template:
    metadata:
      labels: {{- include "redis.labels" . | nindent 8 }}
      {{- if and .Values.metrics.enabled .Values.metrics.podLabels }}
      {{- toYaml .Values.metrics.podLabels | nindent 8 }}
      {{- end }}
```

Apply this to the rest. Notice the `{{-` and the `nindent` in the toYaml command